### ISSUE TITLE
callbacks are not working

### ISSUE THREAD

**Issue by jothikannan** [usablica/intro.js#628]
introJs().onhintsadded(function() {
  alert("all hints were added");
});

introJs().onhintclose(function() {
  alert("hint closed");
});

there callbacks are not working to me [Workout](http://jsfiddle.net/88zodmmv/1/)

**Comment by intellisense** [comment]
Yeah I was having same issue looks like the way to do it is this:

```
var $hints = introJs().addHints();
$hints.onhintclose(function(e) {
  alert("Hint closed");
});
```
